website/app/styles/doc-components/algolia-search/trigger.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+@use "../../typography/mixins" as *;
+
+// ALGOLIA AUTOCOMPLETE SEARCH > BUTTON (TRIGGER)
+
+.aa-DetachedSearchButton {
+  @include doc-font-style-navigation();
+  display: flex;
+  gap: 4px;
+  align-items: center;
+  height: 36px;
+  margin: 0;
+  padding: 4px 12px;
+  color: var(--doc-color-gray-400);
+  line-height: 16px; // override
```

Comment:
I didn't see any effect when I commented out this line-height value in the browser

Reply:
Removed.